fastlane/lib/fastlane/actions/zip.rb

```diff
@@ -40,6 +40,16 @@ def self.available_options
         ]
       end
 
+      def self.example_code
+        [
+          "zip",
+          "zip(
```

Comment:
Based on the code in the `erb` file, I am a bit confused why we would want to have two entries in this example. It looks like the `erb` file is iterating over the things in this list and putting them in that code block, is there a reason why there is `'zip'` and `'zip(...` in there? 

I could be missing something obvious here 😄